tests/xcodeproj/build.sh

```diff
@@ -2,4 +2,6 @@ set -eux
 
 cd $(dirname $0)
 
-xcodebuild -project Single-Application.xcodeproj -quiet
+xcodebuild -project Single-Application-Project.xcodeproj -quiet
+export SIM_DEVICE_ID=`xcodebuild -project Single-Application-Project.xcodeproj -scheme "Single-Application-UnitTests" -showdestinations | grep "platform:iOS Sim" | head -1 | ruby -e "puts STDIN.read.split(',')[1].split(':').last"`
```

Comment:
I will put back this change in later when I need it, right now it is not necessary run these tests. I think it will be necessary when the tests ensure that runs can run, when they are supposed to run inside an apphost.